pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -55,6 +55,11 @@ func CmdCreate() *cobra.Command {
 	cmd.Flags().StringSlice("allowed_buckets", nil,
 		"Set the user allowed buckets list (use commas or multiple flags)")
 	cmd.Flags().String("default_resource", "", "Set the default resource, on which new buckets will be created")
+	cmd.Flags().Bool("nsfs_account_config", false, "This flag is for creating nsfs account")
+	cmd.Flags().Int("uid", -1, "Set the nsfs uid")
+	cmd.Flags().Int("gid", -1, "Set the nsfs gid")
+	cmd.Flags().String("new_buckets_path", "/", "Change the default nsfs mount path")
```

Comment:
change the default nsfs mount path sounds misleading -  maybe keep with Set the new bucket (default) path

Reply:
or maybe explain that it will set the path where new buckets of this account will be created.

Reply:
I am not sure I agree, this is the mount path for the nsfs.

Reply:
this is per account right? not really the mount point